lib/convertor.js

```diff
@@ -71,8 +63,13 @@ class MongoToKnex {
                     return relation.join_table === table;
                 });
 
+                // CASE: assume it's a column on the destination table
                 if (!relation) {
-                    throw new Error('Can\'t find relation in config object.');
+                    return {
+                        column: `${column}`,
```

Comment:
```suggestion
                        column: column,
```
no need for template string here :wink: